lib/still/preprocessor/renderer.ex

```diff
@@ -58,7 +58,7 @@ defmodule Still.Preprocessor.Renderer do
         Module.concat(["R#{Enum.random(0..100_000)}" | [@preprocessor | name]])
       end
 
-      defp create_view_renderer(name, content, metadata) do
+      defp create_template_renderer(name, content, metadata) do
```

Comment:
Not sure about this change. LMK if this function name wasn't supposed to change.

Reply:
I think this change makes sense 👍